README.md

```diff
@@ -194,6 +194,15 @@ export DOCKERIMAGE=condaforge/linux-anvil-aarch64
 bash build_miniforge.sh
 ```
 
+## Rosetta on Mac with Apple Silicon Hardware
+To create new environments that use packages compiled for Intel chips (using rosetta) one should use `CONDA_SUBDIR=osx-64`.
```

Comment:
Shouldn't this be in the conda docs instead of here?

Reply:
Maybe it should go here: https://github.com/conda-forge/conda-forge.github.io/blob/main/src/user/tipsandtricks.rst

Reply:
@hmaarrfk 
I don't mind where I just think that there would be a benefit of documenting it :) The reason why I put it in here is because many M1 users will read some medium articles which typically point to miniforge.

So is tipsandtricks the decision or are there other opinions?